packages/stream_video/lib/src/call/call.dart

```diff
@@ -683,17 +678,15 @@ class Call {
       _credentials = joinedResult.data;
       _previousSession = _session;
 
-      final isWsHealthy = _previousSession?.sfuWS.isConnected ?? false;
-
       final reconnectDetails =
           _reconnectStrategy == SfuReconnectionStrategy.unspecified
               ? null
               : await _previousSession?.getReconnectDetails(_reconnectStrategy);
 
-      if (performingRejoin || performingMigration || !isWsHealthy) {
+      if (performingRejoin || performingMigration || !performingFastReconnect) {
         _logger.v(
           () =>
-              '[join] creating new sfu session (rejoin: $performingRejoin, migration: $performingMigration, wsHealthy: $isWsHealthy)',
+              '[join] creating new sfu session (rejoin: $performingRejoin, migration: $performingMigration, fast: $performingFastReconnect)',
```

Comment:
The original `_reconnectStrategy` is an enum, so isn't this the same as
```dart
if(!performingFastReconnect)
```
The logger also doesn't make sense if I understand it correctly, because `fast` is always `false`.

When `performingRejoin` or `performingMigration` are true `performingFastReconnect` has to be false.